pegasus/sites.v3/code.org/public/music/index.haml

```diff
@@ -0,0 +1,88 @@
+---
+title: Music Lab
+theme: responsive_full_width
+---
+
+- button_url = CDO.studio_url("/s/music-intro-2024")
+- button_string = hoc_s("music_lab.button.try_music_lab")
+
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/page/music-styles.css", rel: "stylesheet"}
+
+%section.hero-banner-basic.overlay
+  .wrapper.grid-container.justify-space-between.align-items-center.gap-4
+    .text-wrapper
+      %p.body-one.white.has-icon
+        =hoc_s("music_lab.introducing_music_lab_strong", markdown: :inline, locals:{music_lab: "Music Lab"})
+      %h1.white
+        =hoc_s("music_lab.banner.heading")
+      %p.body-one.white
+        =hoc_s("music_lab.banner.desc")
+      %a.link-button.white{href: button_url}
+        =button_string
+      .afe.flex-container.align-items-baseline.wrap
+        %p.add-margin-top.no-margin-bottom.white
+          =hoc_s("music_lab.in_partnerhip_with")
+        %figure
+          %img{src: "/images/avatars/amazon_future_engineer_light.png", alt: hoc_s("music_lab.amazon_future_engineer")}
+    %figure
+      %img{src: "/images/music-lab/music-lab-banner-graphic.png", alt: "", style: "width: 100%"}
+
+-# TODO - ADD FINAL LIST OF ARTISTS BEFORE LAUNCH
+%section.bg-neutral-light
+  .wrapper.flex-container.justify-space-between.align-items-center.gap-1.wrap.flex-direction-column-mobile
+    .text-wrapper.flex-1
+      %h2.heading-lg
+        =hoc_s("music_lab.featuring_songs_by")
+      %p.no-margin-bottom
+        =hoc_s("music_lab.artist_list", markdown: :inline, locals:{artist_list: "The Chainsmokers, Rosa Linn, Carly Rae Jepson, Dove Cameron, Aloe Blacc"})
```

Comment:
Should we use [Intl.ListFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat) here?

Reply:
That looks like a JS library; is there a corresponding Ruby library we can use?

Reply:
Kind of, but it seems it's up to us to provide all the templates.  

https://www.rubydoc.info/gems/activesupport/Array:to_sentence

Reply:
That looks really cool. Maybe something @code-dot-org/platform can look at supporting in the future, but I think it's out of scope for this work.